src/utils/geometry.js

```diff
@@ -67,19 +67,12 @@ export function neg (v) {
     return [-v[0], -v[1]];
 }
 
-/**
- * Create a vector which goes from p1 to p2
- */
-function vector (p1, p2) {
-    return [p2[0] - p1[0], p2[1] - p1[1]];
-}
-
 /**
  * Return the vector scaled to length 1
  */
-function normalize (v) {
-    const s = Math.hypot(v[0], v[1]);
-    return [v[0] / s, v[1] / s];
+function normalize (v1, v2) {
```

Comment:
If `v1` and `v2` are `x` and `y` coordinates then `v1` and `v2` are bad names

Reply:
No. v1 and v2 are the start and end of a vector. I borrow the name from the Math.hypo docs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot

Reply:
`v1` and `v2` cannot be vectors because `Math.hypot` receives scalars. Please, review also the usage of this function

Reply:
Yes, they are using those names and `v` comes from `value` if you read their docs, and they have to use numeric values since the function is generic and is not limited to 2D.

In our context, however, we work in 2D and `v1` and `v2` can be confused by `vertex1` and `vertex2`